dist/node/canvg.d.ts

```diff
@@ -1,23 +0,0 @@
-declare namespace canvg {
```

Comment:
any reason the type file was deleted?

Reply:
Fixed.

Am used to `rm -rf dist/ && npm run build` being a sane thing to do, didn't realise there were precious files under there.